### DISCUSSION TITLE
.sh install script for install of frontend and backend

### DISCUSSION THREAD

**Discussion by A-Yamout** [olivia-ai/olivia#152] [Q&A]
I am looking for help with a .sh script that will install Olivia and all of its dependencies onto Linux would nice to have a one-line install, If you want to help with this click on the [LINK!](https://github.com/A-Yamout/Olivia-AI-Install-Script). Any help would be appreciated. Thanks and happy Monday!

**Comment by jusso-dev** [comment]
Any reason you can't simply use the Docker image provided?

**Comment by A-Yamout** [comment]
For some people they don't know how to use any bash and seams easier to install on a Linux server.
